nl.sarine.gpxViewer.json

```diff
@@ -0,0 +1,150 @@
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/clutter/1.26/clutter-1.26.4.tar.xz",
+                    "sha256": "8b48fac159843f556d0a6be3dbfc6b083fc6d9c58a20a49a6b4919ab4263c4e6"
+                }
+            ]
+        },
+        {
+            "name": "clutter-gtk",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/clutter-gtk/1.8/clutter-gtk-1.8.4.tar.xz",
+                    "sha256": "521493ec038973c77edcb8bc5eac23eed41645117894aaee7300b2487cb42b06"
+                }
+            ]
+        },
+        {
+            "name": "libchamplain",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dvapi=true"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libchamplain/0.12/libchamplain-0.12.20.tar.xz",
+                    "sha256": "0232b4bfcd130a1c5bda7b6aec266bf2d06e701e8093df1886f1e26bc1ba3066",
+                    "x-checker-data": {
+                        "type": "gnome",
+                        "name": "libchamplain"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "intltool",
+            "cleanup": [ "*" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+                    "sha256": "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+                }
+            ]
+        },
+        {
+            "name": "gdl",
+            "cleanup" : [
+                "/include",
+                "/lib/pkgconfig"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gdl/3.34/gdl-3.34.0.tar.xz",
+                    "sha256": "858b30f0cdce4c4cb3e8365a7d54ce57c388beff38ea583be5449bc78dda8d02"
+                }
+            ]
+        },
+        {
+            "name": "metainfo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D -m0644 nl.sarine.gpxViewer.metainfo.xml /app/share/metainfo/nl.sarine.gpxViewer.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "nl.sarine.gpxViewer.metainfo.xml"
+                }
+            ]
+        },
+        {
+            "name" : "gpx-viewer",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/DaveDavenport/gpx-viewer"
```

Comment:
this must refer to a specific commit, release or tag.

Reply:
ah yes

Reply:
working on it, let's see if we can tag, if not a commit will work